src/main/scala/temple/generate/server/go/service/main/GoServiceMainHandlersGenerator.scala

```diff
@@ -226,6 +226,64 @@ object GoServiceMainHandlersGenerator {
       },
     )
 
+  private def generateParseTimeBlock(name: String, attributeType: AttributeType): String =
+    mkCode.lines(
+      genDeclareAndAssign(
+        genMethodCall(
+          "time",
+          "Parse",
+          attributeType match {
+            case DateType     => doubleQuote("2006-01-02")
+            case TimeType     => doubleQuote("15:04:05.999999999")
```

Comment:
What do these hard coded values mean?

Reply:
They're layouts according to the reference time of 
`January 2nd at 15:04:05 2006 in timezone +07:00` (1-2-3-4-5-6-7)

https://gobyexample.com/time-formatting-parsing

Reply:
Dumb right